code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1042,7 +1042,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	M.adjustFireLoss(-2)
 	return ..()
 
-/datum/reagent/consumable/ethanol/mushi_kombucha/reaction_mob(mob/living/M, method=TOUCH)
+/datum/reagent/consumable/ethanol/aloe/reaction_mob(mob/living/M, method=TOUCH)
```

Comment:
?

Reply:
Forgot to add this in, will do now. Basically wrong code for aloe's text part.